src/facade/cmd_arg_parser.cc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include "facade/cmd_arg_parser.h"
+
+#include <absl/strings/ascii.h>
+#include <absl/strings/match.h>
+#include <absl/strings/numbers.h>
+
+#include "base/logging.h"
+#include "facade/error.h"
```

Comment:
I moved all function longer than 3 lines and all functions that have some heavy dependencies (abs and out stuff)